src/main/drivers/vtx_common.h

```diff
@@ -62,6 +62,39 @@ typedef enum {
     VTXDEV_UNKNOWN    = 0xFF,
 } vtxDevType_e;
 
+// VTX magic numbers used for spektrum vtx control
+#define VTX_COMMON_BAND_USER      0
+#define VTX_COMMON_BAND_A         1
+#define VTX_COMMON_BAND_B         2
+#define VTX_COMMON_BAND_E         3
+#define VTX_COMMON_BAND_FS        4
+#define VTX_COMMON_BAND_RACE      5
+
+// RTC6705 RF Power index 25 or 200 mW
+#define VTX_6705_POWER_25      1
```

Comment:
6705 is not supported on INAV

Reply:
I removed all of the vtx code with the exception of a few placeholders blocked by `USE_SPEKTRUM_VTX_CONTROL`. If anything, will just make a pull request later on with support for this.